test/test_matcher.py

```diff
@@ -98,15 +106,32 @@ def get_records(num=1):
 output_queue_url = "https://queue.amazonaws.com/aws_account_number/tdr-api-update-intg"
 
 
-def set_environment():
-    os.environ["ENVIRONMENT"] = "intg"
+def encrypt(key, kms, value):
+    return base64.b64encode(kms.encrypt(
+        KeyId=key,
+        Plaintext=bytearray(value, 'utf-8'),
+        EncryptionContext={
+            'LambdaFunctionName': 'test-function-name'
+        }
+    )['CiphertextBlob']).decode('utf-8')
+
+
+def set_environment(kms):
+    key = kms.create_key(
+        Policy='string',
+        Description='string',
+    )['KeyMetadata']['KeyId']
+    print(encrypt(key, kms, "intg"))
+    os.environ["ENVIRONMENT"] = encrypt(key, kms, "intg")  # intg
```

Comment:
Is the comment necessary here? It looks like it was supposed to explain a base64-encoded value, but it's actually a plain text value?

Reply:
Yeah you're right, I tried it with the base64 encoded values but it didn't work so  I changed it to work like this. I've removed the comment.